token-minter/sources/modules/mint_stage.move

```diff
@@ -358,6 +358,17 @@ module minter::mint_stage {
 
     // ====================================== View Functions ====================================== //
 
+    #[view]
+    public fun current_active(collection: Object<Collection>): Option<u64> acquires MintStageData, MintStage {
```

Comment:
`ccurent_active_stage` may be a better name? LMK

Reply:
changed